firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -134,4 +134,10 @@ public boolean clear() {
   private String getSharedPreferencesKey(String key) {
     return String.format("%s|%s", persistenceKey, key);
   }
+
+  @NonNull
+  public boolean isFidRegistered(@Nullable PersistedFidEntry persistedFidEntry) {
```

Comment:
imo this should go on PersistedFidEntry or some sort of static util class... Otherwise it feels like we're just putting it here for convenience

Reply:
Updated. PTAL.